jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py

```diff
@@ -47,6 +48,9 @@ def pachyderm_resources():
 
     yield repos, branches, files
 
+    for repo in repos:
+        client.pfs.delete_repo(repo=pfs.Repo(name=repo))
```

Comment:
Seems like cleaning up the repos after tests run stabilized the CI tests 🤷🏻 This seems like a good change anyways since normally the three repos just exist after test runs which can be confusing.

Reply:
strange, but glad you tracked it down!